### PR TITLE
Add another example to matrix.include

### PR DESCRIPTION
**Pull request by muizzk**
@cotsog please review.

### CODE REVIEW

File: user/build-matrix.md [hunk 1]
```diff
@@ -298,3 +298,27 @@ matrix:
...
+      node_js:
+      - 9
+      script:
+      - node -e "console.log('Hi from NodeJS!')"
+
+    - language: java
+      jdk: openjdk8
+      script:
+      - javac --help
+```
```
Comment (cotsog) [inline user/build-matrix.md]:
````suggestion
```
{: data-file=".travis.yml"}
````

File: user/build-matrix.md [hunk 2]
```diff
@@ -298,3 +298,37 @@ matrix:
...
+```yaml
+dist: xenial
+language: php
+php:
+  - '5.6'
+
+matrix:
+  include:
+    - language: python
+      python:
```
Comment (cotsog) [inline user/build-matrix.md]: please write is as `python: 3.6`

File: user/build-matrix.md [hunk 3]
```diff
@@ -298,3 +298,37 @@ matrix:
...
+      - 3.6
+      script:
+      - python -c "print('Hi from Python!')"
+
+    - language: node_js
+      node_js:
```
Comment (cotsog) [inline user/build-matrix.md]: similarly as above, `node_js: 9`

File: user/build-matrix.md [hunk 4]
```diff
@@ -298,3 +298,37 @@ matrix:
...
+      - node -e "console.log('Hi from NodeJS!')"
+
+    - language: java
+      jdk: openjdk8
+      script:
+      - javac --help
+```
+This creates a build with 3 jobs as follows:
+
+* A PHP 5.6 + Python 3.6 job
```
Comment (cotsog) [inline user/build-matrix.md]: It's not a combination of both, it's only generating:

Python 3.6
Node.js 9
Java 8 job

The web UI is wrong when it prints PHP 5.6 alongside these jobs

File: user/build-matrix.md [hunk 5]
```diff
@@ -298,3 +298,37 @@ matrix:
...
+    - language: node_js
+      node_js:
+      - 9
+      script:
+      - node -e "console.log('Hi from NodeJS!')"
+
+    - language: java
+      jdk: openjdk8
+      script:
+      - javac --help
```
Comment (cotsog) [inline user/build-matrix.md]:
```suggestion
      - javac -help
```

File: user/build-matrix.md [hunk 6]
```diff
@@ -298,3 +298,36 @@ matrix:
...
+    - language: java
+      jdk: openjdk8
+      script:
+      - javac -help
+```
+{: data-file=".travis.yml"}
+This creates a build with 3 jobs as follows:
+
+* A Python 3.6 job
+* A NodeJS 9 job
```
Comment (cotsog) [inline user/build-matrix.md]:
```suggestion
* A  Node.js 9 job
```

File: user/build-matrix.md [hunk 7]
```diff
@@ -298,3 +298,36 @@ matrix:
...
+* A Java8 job
```
Comment (cotsog) [inline user/build-matrix.md]:
```suggestion
* A Java OpenJDK 8 job
```

**Comment by Pezi777** [comment]
thank you so much @muizzk

**Pull request merged**